src/Distribution/Nixpkgs/Nodejs/Cli.hs

```diff
@@ -106,6 +111,10 @@ runConfigParser = RunConfig
   <*> O.switch
       (O.long "offline"
     <> O.help "Makes yarn2nix fail if network access is required")
+  <*> O.optional (O.option O.str
+     (O.long "license-data"
+   <> O.metavar "FILE"
+   <> O.help "Path to a license.json equivalent to lib.licenses from nixpkgs"))
```

Comment:
Should we maybe just statically use the `lib.licenses` from the nixpkgs yarn2nix is built with? That is the easiest I’d assume, and I’m not sure people really need to override it.

Reply:
I patch the source to set a static one in `default.nix`, check one of the later commits.

Reply:
Yeah, but unsure whether I’d provide the argument in the first place, e.g. if it’s worth supporting that as a dynamic feature.